### DISCUSSION TITLE
About CVE-2023-46245

### DISCUSSION THREAD

**Discussion by hack0ps** [kimai/kimai#4558] [General]
Hi all

I'm planning to use Kimai and while doing my research I found this security advisory: https://github.com/kimai/kimai/security/advisories/GHSA-fjhg-96cp-6fcw

It is unclear to me if which versions are affected and whether it has ben mitigated or solved.
If not, is there a known workaround? Is it planned to be fixed in an upcoming release? Or am I just missing something?

Many thanks!

**Comment by kevinpapst** [comment]
Do you think I publish a security advisory without a fix first? That would be awfully irresponsible and stupid. While I agree to be stupid quite often, I disagree on that level of wildness 😁 

Thanks for the question.

I updated the [CVE](https://github.com/kimai/kimai/security/advisories/GHSA-fjhg-96cp-6fcw) with the patched version (didn't knew I could/should do that) and also updated the [release info 2.1.0](https://github.com/kimai/kimai/releases/tag/2.1.0) to make it clearer.

**Reply by hack0ps** [reply]
I have to admit I was quite confused :smile: 
Thanks for clarifying and updating the documentation accordingly!
